skymp5-server/cpp/server_guest_lib/PapyrusObjectReference.cpp

```diff
@@ -443,10 +446,15 @@ VarValue PapyrusObjectReference::MoveTo(
   dest.x += xOffset;
   dest.y += yOffset;
   dest.z += zOffset;
-  const NiPoint3 rotation =
-    matchRotation ? objectReference->GetAngle() : _this->GetAngle();
+  const NiPoint3 rotation = matchRotation ? objectReference->GetAngle()
+                                          : _thisObjectReference->GetAngle();
   LocationalData data{ dest, rotation, objectReference->GetCellOrWorld() };
-  _thisActor->Teleport(data);
+  if (_thisActor) {
+    _thisActor->Teleport(data);
+  } else {
```

Comment:
will never be called because of if (!_thisObjectReference || !objectReference || !_thisActor)